__tests__/article.test.js

```diff
@@ -29,6 +30,38 @@ describe.only('/api/articles/:id', () => {
         });
     });
 
+
+    test('GET: 200  response should return with an array of objects', () => {
+      return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+         const articles = response._body.articles;
+         articles.forEach((article) => {
```

Comment:
As we are using a forEach here - if the array is empty then this test will pass by default as none of the expect statements will be called. A test with no expect statements will pass by default. We should first check that there is something in the array (check its length)